db/migrate/20221122190541_create_comments.rb

```diff
@@ -0,0 +1,13 @@
+class CreateComments < ActiveRecord::Migration[7.0]
+  def change
+    create_table :comments do |t|
+      t.integer :author
+      t.integer :post
+      t.text :text
```

Comment:
- [ ] You really did a great job. Just like you did with author, you have to set post as a foreign key to comment too
i.e (`AddPostsRefToComments`)

Reply:
@m05e5 Thank you for your feedback, I forgot about that 🤭